packages/rpc-core/src/rpc-methods/common.ts

```diff
@@ -24,3 +24,6 @@ export type RpcResponse<TValue> = Readonly<{
     }>;
     value: TValue;
 }>;
+
+// TODO: Eventually move this into whatever package implements transactions
+export type TransactionSignature = string & { readonly __sig: unique symbol };
```

Comment:
Want to throw an assertion function into this PR? Sort of like I did in: https://github.com/solana-labs/solana-web3.js/commit/dfc6d588b4b7c880d74dc727399ee8a84979b480#diff-f50f62f903effcf4d511f3f216959cd18570bfd499b12a824d8fc29effee6172R5-R27

Reply:
Thanks! Added in 1111ae5a0e472350181438cdf0a6c30bd6e1d165